src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java

```diff
@@ -658,6 +658,25 @@ public void shouldGetNextStepsConfirmationForPA17Form() {
 
     }
 
+    @Test
```

Comment:
There is also a ConfirmationResponseServiceFeatureTest, which you should be able to add to, to reduce demands on the FTs